preview-src/index.css

```diff
@@ -384,12 +406,18 @@ body .comment-form .form-actions button {
 }
 
 .comment-body blockquote {
```

Comment:
could you change this to just `blockquote`? https://github.com/xtermjs/xterm.js/pull/1392 has an example of a blockquote being used in somewhere that's not a comment-body

Reply:
👍 Opened the scope so it's for all `blockquote` elements